web/react/components/more_direct_channels.jsx

```diff
@@ -213,7 +220,7 @@ export default class MoreDirectChannels extends React.Component {
         const userEntries = users.map(this.createRowForUser);
 
         if (userEntries.length === 0) {
-            userEntries.push(<li key='no-users-found'>{'No users found :('}</li>);
+            userEntries.push(<div key='no-users-found'>{'No users found :('}</div>);
```

Comment:
Shouldn't this be a tr since it's going to be inside a table? I can never remember nesting rules for HTML.